logos/logos-ui/src/app/shared/components/fun-mode-toggle/fun-mode-toggle.html

```
@@ -0,0 +1,10 @@
+<button
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)